lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -3481,6 +3481,34 @@ bool NonLocalAnchor::updateModule(StringAttr oldMod, StringAttr newMod) {
   return updateMade;
 }
 
+bool NonLocalAnchor::updateModuleAndInnerRef(
+    StringAttr oldMod, StringAttr newMod,
+    const llvm::DenseMap<StringAttr, StringAttr> &renameMap) {
+  auto fromRef = FlatSymbolRefAttr::get(oldMod);
+  SmallVector<Attribute> namepathNew;
```

Comment:
should reserve since you know the length.

Reply:
actually, doesn't namepath return a vector?  Can't you just update-in-place your copy?

Reply:
Fixed.